src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/SessionTemplateRepository.kt

```diff
@@ -93,35 +96,23 @@ interface SessionTemplateRepository : JpaRepository<SessionTemplate, Long> {
   @Modifying
   fun deleteByReference(reference: String): Int
 
-  @Modifying
+  @Modifying(clearAutomatically = true)
```

Comment:
have not used clearAutomatically = true before

Reply:
tests were failing as the entity was not being refreshed after updating name.